data/transactions/logic/doc.go

```diff
@@ -191,13 +191,13 @@ var opDocByName = map[string]string{
 	"itxn_field":  "set field F of the current inner transaction to A",
 	"itxn_submit": "execute the current inner transaction group. Fail if executing this group would exceed the inner transaction limit, or if any transaction in the group fails.",
 
-	"box_create":  "make a box",
-	"box_extract": "read from a box",
-	"box_replace": "write to a box",
-	"box_del":     "delete a box",
-	"box_len":     "length of a box",
-	"box_get":     "full contents of a box",
-	"box_put":     "write contents of box",
+	"box_create":  "create a box named A, of length B. Fail if A is empty or B exceeds 32,384.",
```

Comment:
`box_create` appears also to leave result on the stack indicating whether the box already existed

Reply:
correct, I forgot to go back and change. thanks